packages/support/cldr-data/src/install.mjs

```diff
@@ -14,7 +14,7 @@ let coverage, parentPackage, peerPackages, srcUrl;
 import { fileURLToPath } from 'url';
 import { promisify } from 'util';
 import { readPackage } from 'read-pkg';
-import { readPackageUp } from 'read-pkg-up';
+import { readPackageUp } from 'read-package-up';
```

Comment:
Moving to `read-pkg-up` to `read-package-up` as the former package is renamed/deprecated.